tools/setup-dev.sh

```diff
@@ -227,6 +227,13 @@ if ! ${skip_install_deps} ; then
         echo "Dependency installation failed"
         exit 1
       }
+      echo "=> make sure user in libvirt and starting libvirtd"
+      USER=`whoami`
+      sudo usermod -a -G libvirt $USER
+      sudo mv /etc/libvirt/libvirtd.conf /etc/libvirt/libvirtd.conf.bak
+      sudo sed -e 's/#unix_sock_\([g|r]\)/unix_sock_\1/' /etc/libvirt/libvirtd.conf.bak > /etc/libvirt/libvirtd.conf
+      sudo systemctl restart libvirtd
+
```

Comment:
Looks like this does not change anything

Reply:
I just finished a fresh install and it seems to be working fine for me

Reply:
see the logs, it is not working on remote machines, same as before...

Reply:
this code is breaking automation now, it just hangs after the 'echo' message.

Reply:
If I run this manually and press Ctrl+D I'm getting this:
```
./tools/setup-dev.sh: line 236: /etc/libvirt/libvirtd.conf: Permission denied
Submodule 'ceph.git' (https://github.com/aquarist-labs/ceph.git) registered for path 'src/gravel/ceph.git'
Cloning into '/home/ubuntu/aquarium/src/gravel/ceph.git'...
```

Reply:
I dropped your overall changes because it appears they are irrelevant to the issue, the real reason for these errors:
```
02:02:24 +++  ./tools/aqrdev start foo --conservative
02:02:24 >>> Bringing machine 'node1' up with 'libvirt' provider...
02:02:32 >>> Bringing machine 'node2' up with 'libvirt' provider...
02:02:32 EEE Error while connecting to Libvirt: Error making a connection to libvirt URI qemu:///system?no_verify=1&keyfile=/home/ubuntu/.ssh/id_rsa:
02:02:33 EEE Call to virConnectOpen failed: Failed to connect socket to '/var/run/libvirt/libvirt-sock': Permission denied
```

is that the old ssh connection was remaining for the user as it was before libvirt setup, and reconnecting the ssh after setup-dev.sh resolves the problem.